frontend/public/locales/ja/workload.json

```diff
@@ -0,0 +1,127 @@
+{
+  "Value hidden": "非表示の値",
+  "Save file": "ファイルの保存",
+  "binary data": "バイナリーデータ",
+  "No value": "値なし",
+  "Hide values": "値を非表示にする",
+  "Reveal values": "値を表示する",
+  "Data": "データ",
+  "Name": "名前",
+  "Namespace": "namespace",
+  "Size": "サイズ",
+  "Created": "作成",
+  "ConfigMap details": "ConfigMap の詳細",
+  "Binary data": "バイナリーデータ",
+  "CronJob details": "CronJob の詳細",
+  "Schedule": "スケジュール",
+  "Concurrency policy": "コンカレンシーポリシー",
+  "Starting deadline seconds": "開始の期限 (秒)",
+  "second_0": "秒",
+  "Not configured": "未設定",
+  "Last schedule time": "最終スケジュール時間",
+  "Job details": "ジョブの詳細",
+  "Desired completions": "必要なレベルの完了",
+  "Parallelism": "並列処理",
+  "Active deadline seconds": "有効な期限 (秒)",
+  "Containers": "コンテナー",
+  "Current count": "現在のカウント",
+  "Desired count": "必要なカウント",
+  "DaemonSet details": "DaemonSet の詳細",
+  "Volumes": "ボリューム",
+  "Status": "ステータス",
+  "Labels": "ラベル",
+  "Pod selector": "pod セレクター",
+  "{{currentNumber}} of {{desiredNumber}} pods": "{{currentNumber}}/{{desiredNumber}} pod",
+  "DaemonSets": "DaemonSet",
+  "Start rollout": "ロールアウトの開始",
+  "Resume rollouts": "ロールアウトの再開",
+  "Pause rollouts": "ロールアウトの一時停止",
+  "Latest version": "最新バージョン",
+  "Message": "メッセージ",
+  "Update strategy": "更新ストラテジー",
+  "Timeout": "Timeout",
```

Comment:
There are many "Timeout" and "timeout" in this pr are not translated into Japanese and Chinese, if they need to be translated?

Reply:
It looks like this instance is now translated.